membrane_curvature/base.py

```diff
@@ -54,15 +56,42 @@ class MembraneCurvature(AnalysisBase):
     results.gaussian_curvature : ndarray
         Gaussian curvature associated to the surface.
         Arrays of shape (`n_frames`, `n_x_bins`, `n_y_bins`)
+    results.average_z_surface : ndarray 
+        Average of the array elements in `z_surface`. 
+        Each array has shape (`n_x_bins`, `n_y_bins`)
+    results.average_mean_curvature : ndarray 
+        Average of the array elements in `mean_curvature`.
+        Each array has shape (`n_x_bins`, `n_y_bins`)
+    results.average_gaussian_curvature: ndarray 
+        Average of the array elements in `gaussian_curvature`.
+        Each array has shape (`n_x_bins`, `n_y_bins`)
+
+
+    Notes
+    -----
+    The derived surface and calculated curvatures are available in the
+    ``.results``::atttributes. 
+
+    The attribute :attr:`MembraneCurvature.z_surface`
+    contains the derived surface averaged over the `n_frames` of the trajectory.
+
+    The attributes :attr:`MembraneCurvature.mean_curvature` and
+    :attr:`MembraneCurvature.gaussian_curvature` contain the computed values of
+    mean and Gaussian curvature averaged over the `n_frames` of the trajectory.
 
-    Returns
-    -------
-    average_z_surface : ndarray
-        Average of the array elements in `z_surface`
-    average_mean_curvature :
-        Average of the array elements in `mean_curvature`
-    average_gaussian_curvature:
-        Average of the array elements in `gaussian_curvature`
+    Example
+    -----------
+    You can pass a universe containing your selection of reference:
```

Comment:
Needs double colon to format as code
```suggestion
    You can pass a universe containing your selection of reference::
```